os/drivers/Kconfig

```diff
@@ -62,6 +63,7 @@ config DRVR_INVALIDATE
 endif # DRVR_WRITEBUFFER || DRVR_READAHEAD
 
 endmenu # Buffering
+endif # SCHED_WORKQUEUE
```

Comment:
Only DRVR_WRITEBUFFER uses SCHED_WORKQUEUE, DRVR_READHEAD doesn't. Let's move this as a dependancy of DRVR_WRITEBUFFER.